src/neptune_scale/cli/sync.py

```diff
@@ -78,6 +96,8 @@ def start(
         self,
     ) -> None:
         self._log_seq_id_range = self._operations_repository.get_sequence_id_range()
+        self._file_upload_request_init_count = self._operations_repository.get_file_upload_requests_count()
```

Comment:
Pls initialize this in init + mark it as optional + handle its optionality in wait